versions.tf

```diff
@@ -1,12 +1,10 @@
 terraform {
-  required_version = ">= 0.14.11"
+  required_version = ">= 0.15.00"
 
   required_providers {
     aws = {
-      source = "hashicorp/aws"
-      # retrieve launch template by ID starts at 3.21.0
-      # update_config starts at 3.56
-      version = ">= 3.56"
+      source  = "hashicorp/aws"
+      version = ">= 4.0"
```

Comment:
Why is this bumped?

Reply:
That versions probably overly loose as the underlying api-sdk only supports windows from the December release https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#4480-december-19-2022
So 4.48.0